lib/python/satos_payload_sdk/app_framework.py

```diff
@@ -371,8 +371,7 @@ def start_sequence(self, seq_id, seq_params, seq_deadline):
 
         with self.lock:
```

Comment:
We should remove the lock related code altogether.

Reply:
One question : If a specific sequence is in progress, then are we allowing user to call same sequence again ? Or we want a scheme where user can call multiple sequences but can not call same sequence again ? Considering design expectation of "Sending a single sequence at the start of AOI and one at the end of AOI", I believe we should bypass different sequence and do not send same sequence.

Reply:
I don't understand why we need lock here. Moreover, why should we restrict call of same sequence twice or call of different sequences concurrently if user wants to schedule them like that for whatever reason. Its users application design to deal with consequences of such scenarios if they build it like that in their task-plan/schedule.